device/transport/mqtt/package.json

```diff
@@ -31,11 +31,10 @@
     "build": "tsc",
     "test-min": "nyc mocha --reporter dot --require source-map-support/register test/_*_test.js",
     "test": "nyc mocha --reporter spec --require source-map-support/register test/_*_test.js",
-    "ci": "npm -s run lint && npm -s run build && npm -s run test-min && npm -s run check-cover",
-    "check-cover": "nyc check-coverage --statements 97 --branches 91 --functions 98 --lines 97"
+    "ci": "npm -s run lint && npm -s run build && npm -s run test-min",
```

Comment:
>"ci": "npm -s run lint && npm -s run build && npm -s run test-min", [](start = 4, length = 67)

The comma at the end makes this invalid JSON.  `lerna bootstrap` barfs on this line.  (this is why Horton broke when I re-ran this after fixing the github connection)